src/skin/legacyskinparser.cpp

```diff
@@ -1622,6 +1625,18 @@ QWidget* LegacySkinParser::parseEffectSelector(const QDomElement& node) {
     return pSelector;
 }
 
+QWidget* LegacySkinParser::parseEffectParameterKnobComposed(const QDomElement& node) {
```

Comment:
No, the WEffectParameterKnobComposed constructor needs the EffectsManager* argument.